Autopilot/Src/PID.cpp

```diff
@@ -28,14 +31,21 @@ PIDController::PIDController(float _kp, float _ki, float _kd, float _scale, int1
   last_time = 0;
   last_err = 0;
   last_der = 0;
+
+    HAL_TIM_Base_Start(&TIMER);
```

Comment:
Reminder for myself that I need to add auto-formatting to commits so I don't have to bug people about it.